IKZ_plugin/src/ikz_plugin/movpe/movpe1/growth_excel/parser.py

```diff
@@ -119,15 +117,9 @@ class RawFileMovpeDepositionControl(EntryData):
 
 
 class ParserMovpe1IKZ(MatchingParser):
-    def __init__(self):
-        super().__init__(
-            name='MOVPE 1 Deposition Control IKZ',
-            code_name='MOVPE 1 Deposition Control IKZ',
-            code_homepage='https://github.com/FAIRmat-NFDI/AreaA-data_modeling_and_schemas',
-            supported_compressions=['gz', 'bz2', 'xz'],
-        )
-
     def parse(self, mainfile: str, archive: EntryArchive, logger) -> None:
+        from nomad.search import search, MetadataPagination
```

Comment:
Is there a reason you moved this to inside the parse function? We recently had some issued with these "hidden" imports in the central nomad.